openedx/features/partners/views.py

```diff
@@ -47,9 +47,12 @@ def dashboard(request, slug):
     partner = get_object_or_404(Partner, slug=slug)
     courses = get_partner_recommended_courses(partner.slug, request.user)
     registration_countries = get_registration_countries()
-    return render_to_response('features/partners/dashboard.html', {'recommended_courses': courses,
-                                                                   'slug': partner.slug, 'partner': partner,
-                                                                   'registration_countries': registration_countries, })
+
+    context = {'recommended_courses': courses,
+               'slug': partner.slug, 'partner': partner,
+               'registration_countries': registration_countries, }
```

Comment:
Please line up the closing curly brace with the first character of the line that starts the construct, Please reformat
```
    context = {
        'recommended_courses': courses,
        'slug': partner.slug, 'partner': partner,
        'registration_countries': get_registration_countries(),
    }
```